.pre-commit-config.yaml

```diff
@@ -1,11 +1,20 @@
 #largely copied from https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/ and https://mitelman.engineering/blog/python-best-practice/automating-python-best-practices-for-a-new-project/
 repos:
+    - repo: https://github.com/asottile/seed-isort-config
```

Comment:
Oops actually it looks like my advice was outdated, see [here](https://pycqa.github.io/isort/docs/configuration/pre-commit.html)

Reply:
Aha I just blindly copied without googling, sorry